imod/mf6/model.py

```diff
@@ -292,8 +293,36 @@ def write(
 
     @standard_log_decorator()
     def dump(
-        self, directory, modelname, validate: bool = True, mdal_compliant: bool = False
+        self,
+        directory,
+        modelname,
+        validate: bool = True,
+        mdal_compliant: bool = False,
+        crs: Optional[Any] = None,
     ):
+        """
+        Dump simulation to files. Writes a model definition as .TOML file, which
+        points to data for each package. Each package is stored as a separate
+        NetCDF. Structured grids are saved as regular NetCDFs, unstructured
+        grids are saved as UGRID NetCDF. Structured grids are always made GDAL
+        compliant, unstructured grids can be made MDAL compliant optionally.
+
+        Parameters
+        ----------
+        directory: str or Path
+            directory to dump simulation into.
+        modelname: str
+            modelname, will be used to create a subdirectory.
+        validate: bool, optional
+            Whether to validate simulation data. Defaults to True.
+        mdal_compliant: bool, optional
```

Comment:
It unstacks layers into different variables per layer. Since MDAL doesn't support UGRID (layered) 3D data yet. We do not have to add a version number here, as this very likely only going to be implemented in MDAL when we invest in it. I don't see any other institute/company doing that before.